shell/completion.bash

```diff
@@ -139,17 +139,17 @@ _fzf_handle_dynamic_completion() {
   shift
   orig_cmd="$1"
   orig_var="_fzf_orig_completion_$cmd"
-  orig="${!orig_var##*#}"
+  orig="${!orig_var-##*#}"
```

Comment:
Turns out this has a bug and I'm getting `bash: warning: programmable_completion: ls: possible retry loop`.

```sh
foo='complete -F %s ls #_longopt'
orig_var=foo

echo "${!orig_var##*#}"
  # GOOD: _longopt

echo "${!orig_var-##*#}"
  # BAD: complete -F %s ls #_longopt

# Workaround: split the expression into two
orig="${!orig_var-}"
orig="${orig##*#}"
echo "$orig"
  # GOOD: _longopt
```

Reply:
I'm still getting `bash: warning: programmable_completion: ls: possible retry loop` even with the workaround. I'm only getting it though when  [scop/bash-completion](https://github.com/scop/bash-completion) is sourced.

Reply:
I'm still getting `bash: warning: programmable_completion: ls: possible retry loop` even with the workaround. I'm only getting it though when  [scop/bash-completion](https://github.com/scop/bash-completion) is sourced.

Reply:
This may be the one causing it https://github.com/scop/bash-completion/issues/522

Reply:
nevermind. It is working now with the workaround.